### PR TITLE
Astro CLI does not add variables and connections declared in airflow_settings.yaml to the database fir Airflow 2.0.0

### PR DESCRIPTION
**Pull request by andriisoldatenko**
resolves:
- https://github.com/astronomer/issues/issues/1985
- https://github.com/astronomer/issues/issues/2327
- https://astronomer.zendesk.com/agent/tickets/3500

cc @ernest-kr

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
practically works with 2.0.0 and 1.10.12:
```

```



[!img](https://www.evernote.com/l/AMN5BgeyeAlIQ5zQRUnqaEk4wfl2Oh1rPrAB/image.png)

**Review by kaxil — APPROVED** [review]

**Comment by andriisoldatenko** [comment]
**2.0.0:**

```
Added Connection: my_new_connection
Added Pool: test
Added Variable: my_variable
```


```
astro@d82b3aee72e5:/usr/local/airflow$ airflow connections list
[2021-01-28 13:20:25,853] {plugins_manager.py:286} INFO - Loading 2 plugin(s) took 1.08 seconds
id | conn_id           | conn_type | description | host      | schema  | login | password | port | is_encrypted | is_extra_encrypted | extra_dejson | get_uri
===+===================+===========+=============+===========+=========+=======+==========+======+==============+====================+==============+==========================================
1  | my_new_connection | postgres  | None        | 123.0.0.4 | airflow | user  | pw       | 5432 | True         | True               | {}           | postgres://user:pw@123.0.0.4:5432/airflow

```

File: settings/settings.go [hunk 1]
```diff
@@ -150,11 +185,22 @@ func AddConnections(id string) {
 }
 
 // AddPools  is a function to add Pools from settings.yaml
-func AddPools(id string) {
+func AddPools(id string, airflowVersion uint64) {
 	pools := settings.Airflow.Pools
+	baseCmd := "airflow "
+	if airflowVersion >= AirflowVersionTwo {
+		// Airflow 2.0.0 command
+		//  bassed on https://airflow.apache.org/docs/apache-airflow/2.0.0/cli-and-env-variables-ref.html
```
Comment (adam2k) [inline settings/settings.go:193]: This is a nit:

```Suggestion

		// based on https://airflow.apache.org/docs/apache-airflow/2.0.0/cli-and-env-variables-ref.html
```

**Review by adam2k — APPROVED** [review]
This looks good to me! 🎉 

One small typo in a comment, but everything else looks good.

**Pull request merged**